ingestion/src/metadata/utils/s3_utils.py

```diff
@@ -27,6 +28,12 @@
 logger = utils_logger()
 
 
+def _get_json_text(key: str, text: bytes) -> str:
+    if key.endswith(".gz"):
+        return gzip.decompress(text)
+    return text.decode("utf-8")
+
+
```

Comment:
we can use a common util and utilize same function on both gcs and s3

Reply:
but both are not exactly the same function 😅 need to decode text for s3